plugin/pkg/scheduler/algorithm/predicates/error.go

```diff
@@ -24,19 +24,35 @@ const (
 	memoryResoureceName  string = "Memory"
 )
 
+var (
+	// The predicateName tries to be consistent as the predicate name used in DefaultAlgorithmProvider defined in
+	// defaults.go (which tend to be stable for backward compatibility)
+	ErrDiskConflict              = newPredicateFailureError("NoDiskConflict")
+	ErrVolumeZoneConflict        = newPredicateFailureError("NoVolumeZoneConflict")
+	ErrNodeSelectorNotMatch      = newPredicateFailureError("MatchNodeSelector")
+	ErrPodNotMatchHostName       = newPredicateFailureError("HostName")
+	ErrPodNotFitsHostPorts       = newPredicateFailureError("PodFitsHostPorts")
+	ErrNodeLabelPresenceViolated = newPredicateFailureError("CheckNodeLabelPresence")
+	ErrServiceAffinityViolated   = newPredicateFailureError("CheckServiceAffinity")
+	ErrMaxVolumeCountExceeded    = newPredicateFailureError("MaxVolumeCount")
+	// ErrFakePredicateError is used for test only. The fake predicates returning false also returns error
+	// as ErrFakePredicateError.
+	ErrFakePredicateError = newPredicateFailureError("false")
```

Comment:
also consider s/ErrFakePredicateError/ErrFakePredicate/, the extra error is redundant and doesn't match the others.